.prettierrc

```diff
@@ -1,6 +0,0 @@
-{
-    "trailingComma": "es5",
-    "tabWidth": 4,
-    "semi": false,
-    "singleQuote": true
-}
```

Comment:
Why it got removed?

Reply:
I don't like the work of my prettier. For example, the service/employee.js file is not fixed, I could not understand why.  I wanted to reconfigure prettier and configure it correctly.